server/security/auth.js

```diff
@@ -8,7 +8,7 @@ dotenv.config();
 passport.use(new GoogleStrategy({
     clientID: process.env.GOOGLE_CLIENT_ID,
     clientSecret: process.env.GOOGLE_CLIENT_SECRET,
-    callbackURL: "http://localhost:8080/login-success",
+    callbackURL: "http://ec2-3-129-61-157.us-east-2.compute.amazonaws.com:8080/login-success",  // changed to redirect to deployment
```

Comment:
because this is changing between dev and deployment, it would be an easier fix to toss the dynamic part of the string into your env